### PR TITLE
BUGFIX: FET-177 AFM components deprecation

### PR DESCRIPTION
**Pull request by jtomecek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,13 +34,11 @@ __Make the dropdown work. That is, the dropdown `onChange` should reload the upp
...
 
----
-
 We are looking for re-usable code. You are strongly encouraged to "over-engineer" this to show off your software architecture and designing skills. Assume that this abstract application will be the first component of a large scale application.
 
-### Documentation
+### GoodData.UI Documentation
 
-http://sdk.gooddata.com/gooddata-ui/
+[GoodData.UI Documentation](https://sdk.gooddata.com/gooddata-ui/docs/about\_gooddataui.html)
```
Comment (evgeniabogranskaya) [inline README.md]: Shouldn't the URL be https://sdk.gooddata.com/gooddata-ui/docs/about_gooddataui.html (slash after 'about')?

Comment (jtomecek) [reply README.md]: Correct

File: src/App.js [hunk 2]
```diff
@@ -81,28 +76,31 @@ class App extends Component {
 
     render() {
         const projectId = 'xms7ga4tf3g3nzucd8380o2bev8oeknp';
-        const afm = this.getAfm();
-        const afmAllMonths = this.getAfmForAllMonths();
+        const monthFilter = this.getMonthFilter();
```
Comment (petrjanu) [inline src/App.js]: Maybe better:
```javascript
const filters = [this.getMonthFilter()];
```

**Comment by jtomecek** [comment]
All fixes done

**Pull request merged**